cachito/web/static/api_v1.yaml

```
@@ -545,6 +509,34 @@ paths:
                 error:
                   type: string
                   example: The logs for the Cachito request 1 no longer exist
+  "/content-manifest":
+    get:
+      description: Return the content manifest of the specified requests
+      parameters:
+      - name: requests
+        in: path
+        required: false
+        description: The comma separated IDs of the Cachito requests to retrieve the content manifest for
+        schema:
+          type: string
+          example: 1,2,3
+      responses:
+        "200":
+          description: The content manifest of the Cachito requests
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ContentManifest"
+        "400":
+          description: Any of the input Cachito request id is invalid.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  error:
+                    type: string
+                    example: "Request 1 is in state in_progress, not complete or stale states"
```

```suggestion
                    example: "Request 1 is in state in_progress, not complete or stale"
```
Suggested to match the actual error message